### ISSUE TITLE
Missing XPI file?

### ISSUE THREAD

**Issue by fredpinchon** [bobpeers/Alteryx_SDK_GenerateDateRange#1]
Hi Bob,

I don't see the install file for that macro?
Cheers,
Frederic

**Comment by bobpeers** [comment]
Hi Frederic,

I never uploaded the rest since I was unsure this was useful as you can basically do the same using the Generate rows tool. I can upload it if you think it might be useful.

Cheers,
Bob

**Comment by fredpinchon** [comment]
That’s why I was curious to see how that was different from the Generate
Rows tool. If you tell me it is moot, then I’m good, thanks!



*From:* Bob Peers <notifications@github.com>
*Sent:* Wednesday, February 3, 2021 23:18
*To:* bobpeers/Alteryx_SDK_GenerateDateRange <
Alteryx_SDK_GenerateDateRange@noreply.github.com>
*Cc:* fredpinchon <frederic@lakeofdata.com>; Author <
author@noreply.github.com>
*Subject:* Re: [bobpeers/Alteryx_SDK_GenerateDateRange] Missing XPI file?
(#1)



Hi Frederic,

I never uploaded the rest since I was unsure this was useful as you can
basically do the same using the Generate rows tool. I can upload it if you
think it might be useful.

Cheers,
Bob

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/bobpeers/Alteryx_SDK_GenerateDateRange/issues/1#issuecomment-773090205>,
or unsubscribe

**Comment by bobpeers** [comment]
It was just more specific so it used a calendar selector for the dates which is nicer to use :)
![image](https://user-images.githubusercontent.com/4363445/106858592-31324a00-66c2-11eb-8b77-ab8a1b1ffa4e.png)
